app/frontend/errors/errorMessages.ts

```diff
@@ -87,6 +87,14 @@ const internalErrorMessages: {[key in InternalErrorReason]: (params?: any) => st
     'ServerError: Our servers are probably down. Please try again later and if the problem persists, contact us.',
   [InternalErrorReason.EpochBoundaryUnderway]: () =>
     'Our servers are temporarily down while Cardano is undergoing an epoch boundary. We should be back in a few minutes.',
+  [InternalErrorReason.BitBox02OutdatedFirmwareError]: ({message}) =>
+    `BitBox02OutdatedFirmwareError: Your BitBox02 firmware (version ${message}) is outdated. Please update your firmware to version ${
+      BITBOX02_VERSIONS[CryptoProviderFeature.MINIMAL].major
+    }.${BITBOX02_VERSIONS[CryptoProviderFeature.MINIMAL].minor}.${
+      BITBOX02_VERSIONS[CryptoProviderFeature.MINIMAL].patch
```

Comment:
Not too relevant for this PR, but we have a lot of this duplicated and spacious versioning code which could be quite prettily solved by creating a helper function, maybe we should refactor it in the future.